management/views.py

```diff
@@ -49,7 +49,6 @@ class IndexView(StaffRequiredMixin, TemplateView):
 class CohortListView(StaffRequiredMixin, ListView):
     template_name = 'management/cohort_list.html'
     model = Cohort
-    paginate_by = 50
 
```

Comment:
Just making sure, @mfldavidson, we decided that we didn't want to make this change for any of the other management views? This might have come up in the meeting and I was focusing on something else.